modules/betsy/bc6h.glsl

```diff
@@ -0,0 +1,601 @@
+#[versions]
+
+signed = "#define SIGNED";
+unsigned = "";
+
+#[compute]
+#version 450
+
+#include "CrossPlatformSettings_piece_all.glsl"
+#include "UavCrossPlatform_piece_all.glsl"
+
+#VERSION_DEFINES
+#define QUALITY
```

Comment:
@BlueCube3310 Here

Reply:
Disabling the quality setting gets rid of the artifacts, but it also removes the negative values. Similarly, DirectXTex also seems to clamp the values to 0.

Reply:
I’ll see if we can modify the algorithm to preserve the signed argument

Reply:
@BlueCube3310  What GPU and OS are you using? Are you able to try other GPUs?

Sometimes the artifacts are caused by genuine driver bugs.

Reply:
I'm on Windows 10 64bit, I've tested an RTX 4060TI and a GTX 1660TI on the latest drivers, both exhibit the artifacting.